examples/vite/package.json

```diff
@@ -21,7 +21,7 @@
     "typescript": "^5.3.3",
```

Comment:
extra file above here `._data`

Reply:
removed in [c2b0949](https://github.com/ardriveapp/turbo-sdk/pull/254/commits/c2b0949bd7e438738effe826b1b586e5f1271c5f)